# HG changeset patch
# User Rik <rik@octave.org>
# Date 1377712669 25200
#      Wed Aug 28 10:57:49 2013 -0700
# Node ID af49b1385b26361f28c60288d90211d56c82ecee
# Parent  687872d279be8388083431d73416edd2c254841e
build: Fix man pages not be installed with 'make install'

* doc/interpreter/Makefile.am: Use Automake build rule dist_man_MANS to both
build/install man pages and distribute them in source tarball.

diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -25,17 +25,17 @@ TEXINFO_TEX = ../texinfo.tex
 ## Leading PATH_SEPARATOR required due to weak parsing by dvips (12/04/09)
 TEXINPUTS := "$(PATH_SEPARATOR)$(srcdir)$(PATH_SEPARATOR)$(TEXINPUTS)$(PATH_SEPARATOR)"
 export TEXINPUTS
 
 ## Include custom texmf.cnf necessary to run @seealso macro 
 TEXMFCNF := "..$(PATH_SEPARATOR)$(srcdir)/..$(PATH_SEPARATOR)$(TEXMFCNF)$(PATH_SEPARATOR)"
 export TEXMFCNF
 
-dist_man1_MANS = \
+dist_man_MANS = \
   mkoctfile.1 \
   octave-config.1 \
   octave.1
 
 ## The following files are included in the manual via the @EXAMPLEFILE macro.
 ## They are dependencies for the documentation.
 ## They must not be distributed from this directory.
 EXAMPLE_FILES = \
